src/Apps/Order/Components/ExpressCheckout/__tests__/ExpressCheckoutUI.jest.tsx

```diff
@@ -128,6 +145,21 @@ describe("ExpressCheckoutUI", () => {
     expect(env.mock.getAllOperations()).toHaveLength(0)
   })
 
+  it("tracks an express checkout viewed event", () => {
+    renderWithRelay({
+      Order: () => ({ ...orderData }),
+    })
+
+    expect(trackEvent).toHaveBeenCalledWith({
+      action: "expressCheckoutViewed",
+      context_page_owner_id: "a5aaa8b0-93ff-4f2a-8bb3-9589f378d229",
+      context_page_owner_slug: "artwork-slug",
+      context_page_owner_type: "orders-shipping",
+      flow: "Buy now",
```

Comment:
should we update the owner ID? Maybe both of these up in the mock could be like `artwork-slug-from-context` (not sure if this is helpful, but to make clear in the test file where the value comes from)